models/model.js

```diff
@@ -42,6 +43,21 @@ const readArticles = () => {
     });
 };
 
+const createComment = (article_id, username, body) => {
+  if (!Number.isInteger(Number(article_id))) {
+    return Promise.reject({ status: 400, msg: 'Invalid article_id data type' });
+  }
 
+  if (!username || !body) {
+    return Promise.reject({ status: 400, msg: 'Bad Request: Missing properties' });
+  }
```

Comment:
This can be caught as a psql error by your error handler rather than explicitly handled here.